curriculum/challenges/english/01-responsive-web-design/css-box-model/part-007.md

```diff
@@ -0,0 +1,53 @@
+---
+id: 60a3e3396c7b40068ad69970
+title: Part 7
+challengeType: 0
+dashedName: part-7
+---
+
+# --description--
+
+Before you can start styling the `div` you added, you need to link your CSS to your HTML.
+
+Add a `link` element to link your `styles.css` file. Set the `href` to `./styles.css`, and remember to set the `rel` attribute to `stylesheet`.
+
+# --hints--
+
+Your code should have a `link` element.
+
+```js
+assert(document.querySelectorAll('link').length)
```

Comment:
**This lesson wasn't passing for me. I added the `link` element, but it's not getting added to the html...**

<img width="1077" alt="Screen Shot 2021-07-21 at 5 13 38 PM" src="https://user-images.githubusercontent.com/20648924/126566903-33e8cafd-85d3-46a3-a097-cc45eacaa411.png">


**If I duplicate the element, it shows up:**

![Screen Shot 2021-07-21 at 5 13 56 PM](https://user-images.githubusercontent.com/20648924/126566923-a736a5a7-ec00-468c-99cd-3cc0c5d4c619.png)

😕 I tested in firefox and safari - rebased and tested again. All the same for me. I made some regex suggestions that should work for now - so we wouldn't need to rely on that getting rendered. I also added a test and reworded a few of them. Any idea what's going on here @nhcarrigan? I assume it was passing for you at one point. Maybe it's just something on my end.

```suggestion
assert(/<link/.test(code))
```

Reply:
Ah right, we made changes to the renderer to strip out these link elements.